etc/nsjail/sandbox.cfg

```diff
@@ -0,0 +1,144 @@
+name: "compiler explorer user execution sandbox"
+
+mode: ONCE
+hostname: "ce"
+
+time_limit: 0
+max_cpus: 1
+
+log_level: FATAL
+
+rlimit_as: 102400 # 100 GiB
+rlimit_cpu_type: SOFT
+rlimit_fsize: 16 # 16 MiB
+rlimit_nofile: 10
+
+uidmap {
+    inside_id: "0"
+}
+
+gidmap {
+    inside_id: "0"
+}
+
+# must run following as root during system startup
+# cgcreate -a ubuntu:ubuntu -g memory,pids,cpu,net_cls:ce-sandbox
+cgroup_mem_parent: "ce-sandbox"
+cgroup_pids_parent: "ce-sandbox"
+cgroup_net_cls_parent: "ce-sandbox"
+cgroup_cpu_parent: "ce-sandbox"
+
+cgroup_mem_max: 209715200 # 200 MiB
+cgroup_pids_max: 4
+cgroup_cpu_ms_per_sec: 500
+
+mount {
+    src: "/lib"
+    dst: "/lib"
+    is_bind: true
+}
+
+mount {
+    src: "/usr/lib"
+    dst: "/usr/lib"
+    is_bind: true
+}
+
+mount {
+    src: "/lib64"
+    dst: "/lib64"
+    is_bind: true
+    mandatory: false
+}
+
+mount {
+    src: "/lib32"
+    dst: "/lib32"
+    is_bind: true
+    mandatory: false
+}
+
+mount {
+    dst: "/tmp"
+    fstype: "tmpfs"
+    options: "size=20971520,nr_inodes=100" # 20 MiB
+    rw: true
+    noexec: true
+    nodev: true
+    nosuid: true
+}
+
+mount {
+    dst: "/dev"
+    fstype: "tmpfs"
+}
+
+mount {
+    src: "/dev/null"
+    dst: "/dev/null"
+    rw: true
+    is_bind: true
+}
+
+mount {
+    src: "/dev/zero"
+    dst: "/dev/zero"
+    is_bind: true
+}
+
+mount {
+    src: "/dev/urandom"
+    dst: "/dev/random"
+    is_bind: true
+}
+
+mount {
+    src: "/dev/urandom"
+    dst: "/dev/urandom"
+    is_bind: true
+}
+
+mount {
+    dst: "/proc"
+    fstype: "proc"
+}
+
+mount {
+    src: "/opt/compiler-explorer"
+    dst: "/opt/compiler-explorer"
+    is_bind: true
+}
+
+# TODO: review seccomp audit logs
+# this may be better off as a default allow policy with specific syscalled denied
+seccomp_string: "ALLOW {"
```

Comment:
wowser... we may want to build this cfg in the Makefile using some crappy templating if it ever needs editing :)

Reply:
As per slack, will comment this out in a followup change. Thank @apmorton